ofrak_core/ofrak/__main__.py

```diff
@@ -1,7 +1,13 @@
 import sys
 
-from ofrak.ofrak_cli import OFRAKCommandLineInterface
+from ofrak.cli.command.deps import DepsSubCommand
+from ofrak.cli.command.identify import Identify
+from ofrak.cli.command.list import ListSubCommand
+from ofrak.cli.command.unpack import Unpack
+from ofrak.cli.ofrak_cli import OFRAKCommandLineInterface
 
 if __name__ == "__main__":
-    ofrak_cli = OFRAKCommandLineInterface()
+    ofrak_cli = OFRAKCommandLineInterface(
+        (ListSubCommand(), DepsSubCommand(), Identify(), Unpack())
```

Comment:
Aren't all of these subcommands? For consistency, it seems that either all (or none) should have subcommand in the name.